app/experimenter/experiments/api/v6/serializers.py

```diff
@@ -92,8 +98,16 @@ class Meta:
         )
 
     def get_application(self, obj):
+        return obj.application
+
+    def get_app_name(self, obj):
+        return obj.application
+
+    def get_app_id(self, obj):
         if obj.is_fenix_experiment:
-            return obj.channel
+            if obj.channel in NimbusExperiment.FENIX_APP_ID:
+                return NimbusExperiment.FENIX_APP_ID[obj.channel]
+            return ""
```

Comment:
Firefox Desktop is supposed to be `firefox-desktop` as per https://jira.mozilla.com/browse/EXP-825

Reply:
Yep it is:

https://github.com/mozilla/experimenter/pull/4487/files#diff-cc65e5b3a8e25964001f8848e456a4ffea8cfe14059fc7bf96fe133fdbd173efR42

https://github.com/mozilla/experimenter/blob/869d784733ad5dd50ca5caef7b1bd0773f2a5373/app/experimenter/experiments/constants/nimbus.py#L92

Reply:
Ah sorry was reading this wrong